analyzers/rolaguard_base_analyzer/BaseAnalyzerMain.py

```diff
@@ -165,16 +165,17 @@ def process_packet(packet, policy):
 
                     device_session.reset_counter += 1
         
-        elif packet.f_count <= device_session.up_link_counter:
-
-            # Make sure we have processed at least one packet for this device in this run before firing the alarm
```

Comment:
Don't leave comments and explanations out

Reply:
Comments added